packages/slate-editor/src/extensions/web-bookmark/components/WebBookmarkMenu.tsx

```diff
@@ -42,10 +43,22 @@ const LAYOUT_OPTIONS: OptionsGroupOption<BookmarkCardLayout>[] = [
     },
 ];
 
+const PRESENTATION_OPTIONS: OptionsGroupOption<BookmarkNode.Presentation>[] = [
+    {
+        value: BookmarkNode.Presentation.BOOKMARK,
+        label: 'Bookmark',
+    },
+    {
+        value: BookmarkNode.Presentation.EMBED,
+        label: 'Embed',
+    },
+];
```

Comment:
The order of these options should always match in all menus. It has to be: `Embed`, `Bookmark`, `Link` (when it's available)